react/week1/todolist/src/App.css

```diff
@@ -0,0 +1,38 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
+  height: 40vmin;
+  pointer-events: none;
+}
```

Comment:
it's a good practice to clean the `App.css` default css code since we would not use it and we will have our own styling for our application. but if you think you are going to need it then you can just leave it and use it as it is.